metrics/utils/azure/azure_utils.py

```diff
@@ -0,0 +1,35 @@
+import pandas as pd
```

Comment:
Nit. Please add the license header.
It would be cool if we could have some automation to detect/apply changes automatically (similar to what we already do in Java).

Reply:
Adding it to my todo list.